client/client.cpp

```diff
@@ -40,36 +41,32 @@ class ClientHandler {
 		auto protocol = std::make_shared<TBinaryProtocol>(transport);
 		RPCClient client(protocol);
 
-		try {
-			transport->open();
+		transport->open();
 
-			std::string val;
-			// If seed data exists, run the client with data
-			if (seed_data.is_open()) {
-				std::string line;
-				while (std::getline(seed_data, line)) {
-					Operation op = getSeedOperation(line);
-					auto start = high_resolution_clock::now();
-          client.access(val, op);
-          auto end = high_resolution_clock::now();
-					latencies.push_back(duration_cast<microseconds>(end - start).count());
-				}
-			} 
-			// If seed data does not exist, run client on random values
-			else {
-				for (int i = 0; i < 1000; ++i) {
-					Operation op = genRandOperation();
-					auto start = high_resolution_clock::now();
-					client.access(val, op);
-					auto end = high_resolution_clock::now();
-					latencies.push_back(duration_cast<microseconds>(end - start).count());
-				}
+		std::string val;
+		// If seed data exists, run the client with data
+		if (seed_data.is_open()) {
+			std::string line;
+			while (std::getline(seed_data, line)) {
+				Operation op = getSeedOperation(line);
+				auto start = high_resolution_clock::now();
+				client.access(val, op);
+				auto end = high_resolution_clock::now();
+				latencies.push_back(duration_cast<microseconds>(end - start).count());
 			}
+		} 
+		// If seed data does not exist, run client on random values
+		else {
+			for (int i = 0; i < 1000; ++i) {
+				Operation op = genRandOperation();
+				auto start = high_resolution_clock::now();
+				client.access(val, op);
+				auto end = high_resolution_clock::now();
+				latencies.push_back(duration_cast<microseconds>(end - start).count());
+			}
+		}
 
-			transport->close();
-		} catch (TException& tx) {
-			std::cout << "ERROR: " << tx.what() << std::endl;
-  	}
+		transport->close();
 	}
 
 	void getAveLatency() {
```

Comment:
I dislike that you used `Ave` instead of `Avg` for average lol

Reply:
LOL I dislike that you like `Avg` more than `Ave`